pkgs/development/python-modules/backports-zoneinfo/default.nix

```diff
@@ -16,7 +19,17 @@ buildPythonPackage rec {
     sha256 = "sha256-00xdDOVdDanfsjQTd3yjMN2RFGel4cWRrAA3CvSnl24=";
   };
 
-  propagatedBuildInputs = lib.optionals (pythonOlder "3.7") [
+  patches = [
+    (substituteAll {
+      name = "zoneinfo-path";
+      src = ./zoneinfo.patch;
+      zoneinfo = "${pkgs.tzdata}/share/zoneinfo";
```

Comment:
ok so actually the Python `tzdata` package also provides these. I think we should use those instead then, if that package is a hard dependency of this backport.

Reply:
Ah, good call. Updated!